www/pages/brand-assets.tsx

```diff
@@ -8,15 +8,33 @@ import Link from 'next/link'
 import { useRouter } from 'next/router'
 
 import SectionContainer from '~/components/Layouts/SectionContainer'
+import { NextSeo } from 'next-seo'
 
 const { Title, Text } = Typography
 
 const Index = () => {
   // base path for images
   const { basePath } = useRouter()
 
+  const meta_title = 'Branding | Supabase'
+  const meta_description = 'Get Supabase Brand assets here.'
+
   return (
     <>
+      <NextSeo
+        title={meta_title}
+        description={meta_description}
+        openGraph={{
+          title: meta_title,
+          description: meta_description,
+          url: `https://supabase.com/auth`,
+          images: [
+            {
+              url: `https://supabase.com${basePath}/images/product/database/database-og.jpg`,
```

Comment:
use `https://supabase.com/images/og/og-image.jpg` for now 😎

Reply:
You can use this for all the pages for now.

we're working on an [image generator](http://og.supabase.com) though, so we can replace these later